js/adapt-contrib-xapi.js

```diff
@@ -95,9 +96,20 @@ define([
           displayLang: Adapt.config.get('_defaultLanguage'),
           lang: this.getConfig('_lang'),
           generateIds: this.getConfig('_generateIds'),
-          shouldTrackState: this.getConfig('_shouldTrackState')
+          shouldTrackState: this.getConfig('_shouldTrackState'),
+          componentBlacklist: this.getConfig('_componentBlacklist') || ''
         });
 
+        var componentBlacklist = this.get('componentBlacklist');
```

Comment:
Minor: could avoid the if-else below by setting `var componentBlacklist = this.get('componentBlacklist') || '';`

I know this gives an empty string on split (`['']`) but that won't bother the index checking done below to avoid statement sending